### PR TITLE
MINOR: Log member id of the leader when assignment are received

### PR DESCRIPTION
**Pull request by dajac**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by hachikuji — APPROVED** [review]
LGTM

File: core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala [hunk 1]
```diff
@@ -567,7 +567,7 @@ class GroupCoordinator(val brokerId: Int,
 
             // if this is the leader, then we can attempt to persist state and transition to stable
             if (group.isLeader(memberId)) {
-              info(s"Assignment received from leader for group ${group.groupId} for generation ${group.generationId}. " +
+              info(s"Assignment received from leader $memberId for group ${group.groupId} for generation ${group.generationId}. " +
```
Comment (showuon) [inline core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala:570]: nit: Should we wrap the variable with single/double quote to make it clear? Ex: 
`info(s"Assignment received from leader '$memberId' for group '${group.groupId}' for generation '${group.generationId'}. `

**Review by showuon — APPROVED** [review]
LGTM! Left a minor comment. You can decide if you want to update it or not. Thanks.

File: core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala (hunk 1 above)
Comment (dajac) [reply core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala:570]: @showuon At the moment, none of the other log in the group coordinator does this so it feel a bit weird to do it only in this case. I prefer keeping it as it is for now.

Comment (showuon) [reply core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala:570]: No problem. Thank you. :)

**Pull request merged**